template/registry_testfunc.go

```diff
@@ -3,13 +3,16 @@ package template
 import (
 	"fmt"
 	"math"
+	"slices"
 	"strconv"
 
 	"go.expect.digital/mf2"
 	"golang.org/x/text/language"
 )
 
 // RegistryTestFunc is the implementation of the :test:function.
+//
+//nolint:gocognit
 func RegistryTestFunc(name string) func(*ResolvedValue, Options, language.Tag) (*ResolvedValue, error) {
 	if name != "format" && name != "select" {
```

Comment:
Validating "function" is missing here

```suggestion
	if name != "format" && name != "select" && name != "function" {
```

Reply:
:test:function not used anywhere